docs/contrib/Logger.rst

```diff
@@ -259,8 +317,7 @@ call the utl::error/warn with a Tool ID and message ID. For
 compatibility these are defaulted to 'UKN' and '0000' until they are
 updated.
 
-There is no reason to ``puts`` (ie, print) errors in regression tests
-that are caught. The logger prints the error now.
+Do no use "puts", ie, print) errors in regression tests. The logger prints the error now.
```

Comment:
Do not use "puts" (i.e. print)

Reply:
This sentence does not make a lot of sense .
I can't even tell what it is trying to say.
Is it trying to say utl::error should be used instead of 'puts' to report errors in regression tests?
It is completely reasonable to use 'puts' in regression tests.
utl::error stops execution of the test, which may not be what is wanted.
It never makes sense to refer to history in documentation. Who cares what the logger did in the past?

Reply:
Please clarify the correct usage of logger in the regression tests. I have removed historical references.